doc/migrations/v0.46-v1.0.0.md

```diff
@@ -402,7 +403,7 @@ if (await datastore.has(selfKey)) {
 }
 
 const node = await createLibp2p({
-  peerId,
+  peerId
   // ... other options
 })
```

Comment:
these fixes come from pasting this code into a new file in an `aegir` project (helia) and fixing saving causing auto-fix. can revert if desired.